core/src/navigation/ProductSwitcher.html

```diff
@@ -0,0 +1,88 @@
+{#if productSwitcherItems && productSwitcherItems.length &&
+Object.keys(productSwitcherItems[0]).length}
+<div class="fd-shellbar__action fd-shellbar__action--collapsible">
+  <div class="fd-product-switcher">
+    <div class="fd-popover fd-popover--right">
+      <div class="fd-popover__control" on:click="event.stopPropagation()">
+        <button
+          class="fd-button--shell sap-icon--grid"
+          aria-expanded="{dropDownStates.productSwitcherPopover || false}"
+          aria-haspopup="true"
+          on:click="toggleDropdownState('productSwitcherPopover')"
+        ></button>
+      </div>
+      <div
+        class="fd-popover__body fd-popover__body--right"
+        aria-hidden="{!(dropDownStates.productSwitcherPopover || false)}"
+        id="productSwitcherPopover"
+      >
+        <div class="fd-product-switcher__body">
+          <nav>
+            <ul>
+              {#each productSwitcherItems as productSwitcherItem}
+              <li on:click="onActionClick(productSwitcherItem)">
+                <span class="fd-product-switcher__product-icon">
+                  <img src="{productSwitcherItem.icon}" />
+                </span>
+                <span class="fd-product-switcher__product-title">
+                  {productSwitcherItem.label}
+                </span>
+              </li>
+              {/each}
+            </ul>
+          </nav>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+{/if}
+<script type="text/javascript">
+  import { LuigiConfig } from '../services/config.js';
+  import * as Routing from '../services/routing.js';
+  export default {
+    async oncreate() {
+      const productSwitcherConfig = LuigiConfig.getConfigValue(
+        'navigation.productSwitcher'
+      );
+      this.set({
+        config: productSwitcherConfig
+      });
+
+      if (productSwitcherConfig) {
+        const productSwitcherItems = await LuigiConfig.getConfigValueAsync(
+          'navigation.productSwitcher.items'
+        );
+        this.set({
+          productSwitcherItems
+        });
+      }
+    },
+    methods: {
+      onActionClick(productSwitcherItem) {
+        if (productSwitcherItem.externalLink) {
+          window.open(
+            productSwitcherItem.externalLink.url,
+            productSwitcherItem.externalLink.sameWindow ? '_self' : '_blank'
+          );
+        } else {
+          this.goTo(productSwitcherItem.link);
+        }
+      },
+      goTo(path) {
+        this.root.getUnsavedChangesModalPromise().then(() => {
+          Routing.navigateTo(path);
+        });
+      },
+      toggleDropdownState(name) {
+        this.fire('toggleDropdownState', { name });
+        const dropDownStates = this.get().dropDownStates || {};
+      }
+    }
+  };
+</script>
+<style type="text/scss">
```

Comment:
Please change cursor to pointer for all nodes in a product switcher. Additionally, it would be good to add a background colour on hover (as it is in context switcher).